apps/mobile/src/state/chat/atoms/checkAndDeleteOldChatsDataActionAtom.ts

```diff
@@ -0,0 +1,42 @@
+import {type PublicKeyPemBase64} from '@vexl-next/cryptography/src/KeyHolder'
+import {atom} from 'jotai'
+import {focusAtom} from 'jotai-optics'
+import {DateTime} from 'luxon'
+import removeFile from '../../../utils/removeFile'
+import messagingStateAtom from './messagingStateAtom'
+
+const ONE_MONTH_IN_MS = 1000 * 60 * 60 * 24 * 30
+
+export const checkAndDeleteOldChatsDataActionAtom = atom(
```

Comment:
Also would it be possible to remove the inbox also? (only if there is no offer for it nor there is any other open chats)